[

{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#CVE-2020-14343", "@type": ["http://www.w3.org/2002/07/owl#NamedIndividual", "http://docs.oasis-open.org/cti/ns/stix#Vulnerability"], "http://docs.oasis-open.org/cti/ns/stix/core#description": [{"@value": "A vulnerability was discovered in the PyYAML library in versions before 5.4, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. This flaw allows an attacker to execute arbitrary code on the system by abusing the python/object/new constructor. This flaw is due to an incomplete fix for CVE-2020-1747."}], "http://docs.oasis-open.org/cti/ns/stix/core#modified": [{"@value": "2021-03-13T20:12:28.447712Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#created": [{"@value": "2021-03-13T20:12:28.447712Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#spec_version": [{"@value": "2.1"}], "http://docs.oasis-open.org/cti/ns/stix/core#id": [{"@value": "vulnerability--3755142d-a7ba-47fc-a78d-2e0c65657ba6"}], "http://docs.oasis-open.org/cti/ns/stix/core#name": [{"@value": "CVE-2020-14343"}], "http://docs.oasis-open.org/cti/ns/stix/core#created_by_ref": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a"}], "http://docs.oasis-open.org/cti/ns/stix/marking#object_marking_refs": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"}]}

]